Dockerfile

```diff
@@ -49,18 +48,19 @@ RUN \
 # copy sources
 COPY . /home/jhipster/generator-jhipster
 
+# When https://github.com/docker/docker/pull/28499 will be merged and released,
+# remove the following line and add "--chown jhipster:jhipster" to the previous one
 RUN \
   # fix jhipster user permissions
-  chown -R jhipster:jhipster \
-    /home/jhipster \
-    /usr/lib/node_modules && \
+  chown -R jhipster:jhipster /home/jhipster/generator-jhipster
 
+RUN \
   # install jhipster
-  rm -Rf /home/jhipster/generator-jhipster/node_modules \
+  rm -rf /home/jhipster/generator-jhipster/node_modules \
     /home/jhipster/generator-jhipster/yarn.lock \
     /home/jhipster/generator-jhipster/yarn-error.log && \
   su -c "cd /home/jhipster/generator-jhipster && yarn install" jhipster && \
-  su -c "yarn global add file:/home/jhipster/generator-jhipster" jhipster && \
+  yarn global add file:/home/jhipster/generator-jhipster && \
```

Comment:
May I try to put "yarn install" and "yarn add" in the RUN post-USER? or is it useless?